packages/collapse/edit.js

```diff
@@ -1,22 +1,16 @@
 import PropTypes from 'prop-types';
 
 import { Component, Fragment } from '@wordpress/element';
-import { InnerBlocks, PlainText } from '@wordpress/block-editor';
+import { InnerBlocks, PlainText  } from '@wordpress/block-editor';
+import { InspectorControls } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { TextControl } from '@wordpress/components';
```

Comment:
`InspectorControls`, `__` and `TextControl` are unused.

Reply:
Removed in the latest commits